### PR TITLE
LPS-113528 Move `getGeolocation` to frontend-js-web

### PR DESCRIPTION
**Pull request by kresimir-coko**
Alternative to #1992 

With us finally deciding what to do with these kinda things, here's the wanted approach - moving `getGeolocation` to `frontend-js-web`, and removing it from `frontend-js-aui-web`.

### CODE REVIEW

File: modules/apps/frontend-js/frontend-js-web/test/liferay/util/get_geolocation.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import getGeolocation from '../../../src/main/resources/META-INF/resources/liferay/util/get_geolocation';
+
+describe('Liferay.Util.getCropRegion', () => {
```
Comment (diegonvs) [inline modules/apps/frontend-js/frontend-js-web/test/liferay/util/get_geolocation.js:17]:
```suggestion
describe('Liferay.Util.getGeolocation', () => {
```

**Comment by kresimir-coko** [comment]
ci:test:relevant

**Review by bryceosterhaus — APPROVED** [review]
LGTM!

**Comment by kresimir-coko** [comment]
ci:test:relevant

**Comment by kresimir-coko** [comment]
ci:test:relevant

**Comment by kresimir-coko** [comment]
ci:forward

**Pull request closed without merging**